django/contrib/contenttypes/models.py

```diff
@@ -10,13 +10,15 @@
 class ContentTypeManager(models.Manager):
     use_in_migrations = True
 
-    # Cache to avoid re-looking up ContentType objects all over the place.
-    # This cache is shared by all the get_for_* methods.
-    _cache = {}
+    def __init__(self, *args, **kwargs):
+        super(ContentTypeManager, self).__init__(*args, **kwargs)
+        # Cache to avoid re-looking up ContentType objects all over the place.
```

Comment:
I guess "relooking" is okay as a word without a dash, but maybe we could rephrase a bit: Cache shared by all the get_for_* methods to speed up ContentType retrieval.